app/jobs/inherit_permissions_job.rb

```diff
@@ -18,7 +19,13 @@ def perform(work)
 
       # apply the new and deleted attributes
       file.permissions_attributes = attribute_map
-      file.save!
+      persister.save(resource: file)
     end
   end
+
+  private
+
+    def persister
+      Valkyrie::MetadataAdapter.find(:indexing_persister).persister
+    end
```

Comment:
I get that this is shorter than `Valkyrie::MetadataAdapter.find(:indexing_persister).persister`, but repeating in every job seems ugly.  Since most of these jobs inherit from Hyrax::ApplicationJob as a protected method?  Or ideally just create a `change_set_persister` as per my comment above there and have it for every job.